lib/animina/validations/postal_code.ex

```diff
@@ -21,8 +21,8 @@ defmodule Animina.Validations.PostalCode do
       zip_code == nil ->
         :ok
 
-      String.length(zip_code) < 5 || String.length(zip_code) > 5 ->
-        {:error, field: opts[:attribute], message: "length must be 5"}
+      ! String.match?(zip_code, ~r/\A [0-9]{5} \z/x) ->
+        {:error, field: opts[:attribute], message: "must have 5 digits"}
```

Comment:
We need the error messages done with `gettext/1`.

Reply:
I totally agree, but none of the other validations use `gettext`(), and I simply copied from there:

https://github.com/animina-dating/animina/tree/a76efad2931542dc74fd5c530ce3b05213cf5354/lib/animina/validations

I also think that in order to translate properly we need a complete sentence in the validators.

=> #63